src/main/java/com/guciowons/shoppingbasket/Product/Product.java

```diff
@@ -1,18 +1,38 @@
 package com.guciowons.shoppingbasket.Product;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
 import java.math.BigDecimal;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Objects;
 
+@Document
 public class Product {
 
+    @Id
+    private String appId;
     private Integer id;
```

Comment:
To jest jakieś `externalId` czy coś takiego, żeby pokazać, że nie Ty zarządzasz tym polem